src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.tsx

```diff
@@ -62,28 +64,39 @@ export const EditRespondentPopup = ({
     <Modal
       open={popupVisible}
       onClose={handlePopupClose}
-      onSubmit={handleSubmit(submitForm)}
+      onSubmit={isSuccessVisible ? handlePopupClose : handleSubmit(submitForm)}
       title={t('editRespondent')}
-      buttonText={t('save')}
-      hasSecondBtn
+      buttonText={t(isSuccessVisible ? 'ok' : 'save')}
+      hasSecondBtn={!isSuccessVisible}
       onSecondBtnSubmit={handlePopupClose}
       secondBtnText={t('cancel')}
     >
       <StyledModalWrapper>
-        <form onSubmit={handleSubmit(submitForm)} noValidate>
-          <StyledController>
-            <InputController fullWidth name="nickname" control={control} label={t('nickname')} />
-          </StyledController>
-          <StyledController>
-            <InputController
-              fullWidth
-              name="secretUserId"
-              control={control}
-              label={t('secretUserId')}
-            />
-          </StyledController>
-        </form>
-        {error && <StyledErrorText>{getErrorMessage(error)}</StyledErrorText>}
+        {isSuccessVisible ? (
+          <>{t('editRespondentSuccess')}</>
+        ) : (
+          <>
+            <form onSubmit={handleSubmit(submitForm)} noValidate>
+              <StyledController>
+                <InputController
+                  fullWidth
+                  name="nickname"
+                  control={control}
+                  label={t('nickname')}
+                />
+              </StyledController>
+              <StyledController>
+                <InputController
+                  fullWidth
+                  name="secretUserId"
+                  control={control}
+                  label={t('secretUserId')}
+                />
+              </StyledController>
+            </form>
+            {error && <StyledErrorText>{getErrorMessage(error)}</StyledErrorText>}
```

Comment:
StyledErrorText by default has margin-top: -1.5rem.
Is this margin ok here?

Reply:
@IhorNesterenko99  yep